### PR TITLE
Issue/add elastic apm support

### PR DESCRIPTION
**Pull request by Nath82**
#98 Adding elastic apm support to the project to monitor application performance
A second time because I merged the other PR too quickly without waiting for everyone's validation.

### CODE REVIEW

**Comment by Nath82** [comment]
@alextreme @annashamray Sorry for the mess, I saw the green merge button on the other PR and clicked too fast. You can review now.

File: requirements/base.in [hunk 1]
```diff
@@ -24,10 +24,11 @@ mozilla-django-oidc-db
 
 # API libraries
 djangorestframework
-gemma-zds-client<1.0
+gemma-zds-client<1.0P
```
Comment (annashamray) [inline requirements/base.in:27]: Why is this change?

Comment (Nath82) [reply requirements/base.in:27]: Oh, i didn't see it, it should be a miss type, Shift + P instead of Ctrl + P ^^'
Thanks !

**Review by annashamray — APPROVED** [review]

**Review by alextreme — APPROVED** [review]
Thankfully the premature merge wasn't a problem :+1:

**Pull request merged**